packages/office-ui-fabric-react/src/utilities/decorators/withViewport.tsx

```diff
@@ -121,7 +121,7 @@ export function withViewport<TProps extends { viewport?: IViewport }, TState>(
 
```

Comment:
I have trouble with this file, somehow it changed and I cannot revert the change...

Reply:
done changes online, this helps